.isort.cfg

```diff
@@ -0,0 +1,8 @@
+[settings]
+atomic=true
+lines_after_imports=2
+lines_between_types=1
+multi_line_output=5
+not_skip=__init__.py
+
+known_first_party=attr
```

Comment:
Could also go into `setup.cfg`.

Reply:
Hm I used to have issues with it but it seems to work now!